src/app/tasks/task-plagiarism-file-viewer/task-plagiarism-file-viewer.tpl.html

```diff
@@ -1,6 +1,9 @@
 <div class="panel panel-default task-plagiarism-file-viewer">
   <div class="panel-heading">
     <h5 class="panel-title">{{match.username}}: {{match.name}}
+      <!-- <button ng-click="emailStudentsPlagiarism()" class="btn btn-primary">
```

Comment:
Yeah that commented code is safe to be removed, I was just testing that
sorry it slipped through.
On 20 May 2016 5:45 pm, "Andrew Cain" <notifications@github.com> wrote:

In
src/app/tasks/task-plagiarism-file-viewer/task-plagiarism-file-viewer.tpl.html
<https://github.com/doubtfire-lms/doubtfire-web/pull/31#discussion_r64002957>
:

> @@ -1,6 +1,9 @@
>  <div class="panel panel-default task-plagiarism-file-viewer">
>    <div class="panel-heading">
>      <h5 class="panel-title">{{match.username}}: {{match.name}}
> +      <!-- <button ng-click="emailStudentsPlagiarism()" class="btn btn-primary">

Should this be commented out?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/doubtfire-lms/doubtfire-web/pull/31/files/9b4234c78a43eacca5d2eaf3f3ae8b94601e1fb9#r64002957>